abstract-leveldown.js

```diff
@@ -133,6 +133,59 @@ AbstractLevelDOWN.prototype._del = function (key, options, callback) {
   this._nextTick(callback)
 }
 
+AbstractLevelDOWN.prototype.map = function (keys, options, callback) {
+  if (typeof options === 'function') callback = options
+
+  if (typeof callback !== 'function') {
+    throw new Error('map() requires a callback argument')
+  }
+
+  if (Array.isArray(keys) === false || keys === null) {
```

Comment:
```suggestion
  if (!Array.isArray(keys)) {
```

The null-check is redundant.